cognee/tasks/translation/translate_content.py

```
@@ -0,0 +1,705 @@
+
+class GoogleTranslateError(TranslationDependencyError):
+    """GoogleTrans library required."""
+    def __init__(self, message="googletrans is not installed. Please install it with `pip install googletrans==4.0.0-rc1`"):
+        super().__init__(message)
+
+class AzureTranslateError(TranslationDependencyError):
+    """Azure Translate library required."""
+    def __init__(self, message="azure-ai-translation-text is not installed. Please install it with `pip install azure-ai-translation-text`"):
+        super().__init__(message)
+
+class AzureTranslateError(TranslationDependencyError):
```

Ruff (0.12.2)

Redefinition of unused `AzureTranslateError` from line 33

(F811)